src/js/core.js

```diff
@@ -333,15 +333,15 @@
         return !this.options.extensions['imageDragging'];
     }
 
-    function createContentEditable(textarea, doc) {
```

Comment:
Since `createContentEditable` is just an internal function, there shouldn't be a way for anyone to actually call it outside of our internal methods. So we don't need to worry about backward compatibility (if that's your concern) since only our internal code has access to it.

As part of the work to introduce `addElements()` I modified this method to take the `document` in as a param (instead of using `this.options.ownerDocument`) since that was the only time it used `this` as the editor.  However, in order to attach the event handler and call the helper method, it again needed a reference to the editor, so I went back to using `this` as the editor and no longer needed to take the `document` in as a param...that was my thought process at least.